recipes/sccache/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.2.12" %}
+
+package:
+  name: sccache
+  version: {{ version }}
+
+source:
+  url: https://github.com/mozilla/sccache/archive/{{ version }}.tar.gz
+  sha256: 591a82ddbc2e970630a9426c78c25cbc52c3261b06d57cb4e1f11ab8008629fa
+
+build:
+  number: 0
+  missing_dso_whitelist:          # [osx]
+    - /usr/lib/libresolv.9.dylib  # [osx]
+
+requirements:
+  build:
+    - rust
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Comment:
what is the proper way to choose a compiler for windows? I used the `ripgrep` feedstock as example, and it was using this one.

Reply:
This looks fine. If your package can build with msvc instead of the gnu ones you can just use compiler('c')

Reply:
I tried with `compiler('c')`, but then it couldn't link openssl. Reverting to `compiler('m2w64_c')` for now.